app/assets/images/payment_icons/coins.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-coinsph"><title id="pi-coinsph">Coins.ph</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path id="path947" d="M15.14 11.2c.213-.26.427-.52.648-.71.221-.189.45-.308.757-.384.308-.077.694-.112.99-.014.296.098.501.328.65.543.15.215.243.415.29.549.045.133.045.2.045.267" fill="none" stroke="#47baeb" stroke-linecap="square" stroke-linejoin="bevel" stroke-width=".62" paint-order="markers stroke fill"/><path id="path947-1" d="M32.38 11.23c.213-.26.427-.52.648-.71s.45-.307.758-.384c.307-.077.693-.112.99-.014.295.098.5.328.65.543.149.215.242.415.288.549.046.133.046.2.046.267" fill="none" stroke="#47baeb" stroke-linecap="square" stroke-linejoin="bevel" stroke-width=".62" paint-order="markers stroke fill"/><path id="path868-7-5-2" d="M35.76 11.77v3.141" fill="#f0f" stroke="#47baeb" stroke-width=".62"/><ellipse id="path858-9" cx="28.44" cy="12.42" rx="2.539" ry="2.661" fill="#47baeb"/><ellipse id="path862-8" cx="28.46" cy="12.43" rx="1.909" ry="2.009" fill="#fff"/><path id="path22" d="M6.466 13.7a2.447 2.338 0 01-2.756.895 2.447 2.338 0 01-1.702-2.257 2.447 2.338 0 011.761-2.215 2.447 2.338 0 012.732.96l-2.046 1.284z" fill="none" stroke="#51baeb" stroke-width=".586"/><ellipse id="path849" cx="4.46" cy="12.37" rx="2.154" ry="2.041" fill="#fff"/><ellipse id="path858" cx="9.627" cy="12.39" rx="2.539" ry="2.661" fill="#47baeb"/><ellipse id="path862" cx="9.647" cy="12.4" rx="1.909" ry="2.009" fill="#fff"/><path id="path868" d="M13.27 9.892v4.975" fill="#47baeb" stroke="#47baeb" stroke-width=".625"/><path id="path868-7" d="M14.89 9.892v4.975" fill="#f0f" stroke="#47baeb" stroke-width=".625"/><circle id="yellow" cx="13.26" cy="8.353" r=".49" fill="#efa023"/><path id="path868-7-5" d="M18.52 11.74v3.141" fill="#f0f" stroke="#47baeb" stroke-width=".62"/><circle id="path931-0" cx="24.43" cy="14.6" r=".447" fill="#51baeb"/><path id="path868-7-3" d="M26.17 9.837v6.878" fill="#f0f" stroke="#47baeb" stroke-width=".625"/><path id="path868-7-3-3" d="M32.14 8.027v6.878" fill="#f0f" stroke="#47baeb" stroke-width=".625"/><path id="path1069" d="M22.83 10.77c-.41-.358-.822-.716-1.276-.797-.455-.081-.953.115-1.271.374-.319.26-.457.581-.383.91.074.33.357.661.825.857.467.195 1.117.254 1.561.499.444.244.682.673.607 1.07-.076.396-.465.76-.91.93-.444.17-.944.147-1.344-.008-.4-.156-.7-.443-1.002-.73" fill="none" stroke="#47baeb" stroke-width=".62"/></svg>
```

Comment:
Can we remove all `id` attributes except the `<title id="pi-coinsph">`? They are not used.

Reply:
Removed. Rebased.